package.json

```diff
@@ -9,7 +9,10 @@
     "truffle-migrate": "truffle migrate",
     "truffle-compile": "truffle compile",
     "truffle-test": "truffle test",
-    "ganache": "ganache-cli -d"
+    "ganache": "ganache-cli -d",
+    "web:start": "cd ./web && npm start",
+    "web:test": "cd ./web && npm test",
+    "clean": "rm -r ./web/src/contracts"
```

Comment:
I'm cool with these. This might be out of scope, but how do you feel about using yarn?

Reply:
I would love to try it out. What differentiates it from npm?

Reply:
yarn is one extra command, npm is a bit verbose. 
yarn is supposedly faster? Idk it may not be anymore. Yarn does a bit of extra locking, but I keep finding myself using yarn instead of npm.

Lets move to yarn, but not in this PR